src/patterns/dosdp-patterns/upper_tooth_pattern.yaml

```
@@ -18,53 +25,63 @@ pattern_iri: http://purl.obolibrary.org/obo/uberon/patterns/upper_teeth.yaml
 description: "This defines upper teeth."
 
 classes:
-  bone of jaw: 'UBERON:0012360'  # Add ID for bone
-  dentition: 'UBERON:0003672'
-  calcareous tooth: 'UBERON:0001091'
-  in_lateral_side_of: 'BSPO:0000126'
-
+  bone_of_jaw: 'UBERON:0012360'  # Add ID for bone
+  dentition_type: 'UBERON:0003672'
+  calcareous_tooth: 'UBERON:0001091'
 
 relations:
   part_of: 'BFO:0000050'
 
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+  xref: oio:hasDbXref
+
 vars:
-  bone: "'bone of jaw'"  # Could be more specific: "'mandible' or 'premaxilla'" if dosdp_tools can read the OR. # Check with Jim Baloff
+  jaw: 'bone_of_jaw'
+  # Could be more specific: "'mandible' or 'premaxilla'" if dosdp_tools can read the OR. # Check with Jim Baloff
   # premaxilla UBERON_0002244
   # maxilla UBERON:0002397
-  dentition_type: "'dentition'"
+  dentition: 'dentition_type'
   # primary UBERON:0007116
   # secondary UBERON:0007774
-  tooth: "'calcareous tooth'"
+  tooth: 'calcareous_tooth'
   # incisor UBERON_0001098
   # canine UBERON:0003674
   # molar UBERON:0003655
   # premolar UBERON:0007120
-
-data_vars:
-  number: 'xsd:integer'
-  # 1-6
+  syn: "xsd:string"
```

```suggestion
  your_syn: "xsd:string"
```
This is to fix an error.